pkg/controller/cluster/http_handlers.go

```diff
@@ -188,8 +196,114 @@ func (c *Controller) GetenvHandler(w http.ResponseWriter, r *http.Request) {
 
 // CRDConversionHandler - POST /webhook/v1/crd-conversion
 func (c *Controller) CRDConversionHandler(w http.ResponseWriter, r *http.Request) {
-	body, _ := ioutil.ReadAll(r.Body)
 
-	log.Println("conversation")
-	fmt.Println(string(body))
+	dec := json.NewDecoder(r.Body)
+
+	var req apiextensionsv1.ConversionReview
+
+	if err := dec.Decode(&req); err != nil {
+		log.Println(err)
+		w.WriteHeader(500)
```

Comment:
```suggestion
		http.Error(w, http.StatusInternalError, err.Error())
		return
```

Reply:
ready! all errors return with the suggestion